### ISSUE TITLE
With the latest BigSur update  (11.3 20E232) hs.eventtap.keyStrokes no longer works on web forms in password protected fields

### ISSUE THREAD

**Issue by sfosdal** [Hammerspoon/hammerspoon#2858]
This works in a text editor, but not if the cursor focus is in a password protected HTML input box.

```
hs.eventtap.keyStrokes("foo")
```

I suspect Apple made this change intentional, but I am not sure how.

**Comment by slippycheeze** [comment]
I wouldn't be surprised if it was related to "secure event input"; try it in Terminal.app with "secure keyboard" enabled.

**Comment by sfosdal** [comment]
yeah, in Terminal.app, with Secure Keyboard Entry enabled, it fails as well

**Comment by sfosdal** [comment]
Probably for the best.... :/

**Comment by MichaelBaiJie** [comment]
The interesting thing is that hotkey.bind works.  It would be nice to still be able to use modal...

**Comment by sfosdal** [comment]
However [Alfred](https://www.alfredapp.com/) is handling it does seem to work.  Odd
